src/test/scala/com/tresys/nitf/TestNTBBaseline.scala

```diff
@@ -23,6 +23,11 @@ class TestNTBBaseline {
   @Test def test_i_3025b() { runner.runOneTest("i_3025b") }
   @Test def test_i_3034c() { runner.runOneTest("i_3034c") }
   @Test def test_i_3034f() { runner.runOneTest("i_3034f") }
+  @Test def test_i_3034f() { runner.runOneTest("i_3034f_decimal") }
+  @Test def test_i_3034f() { runner.runOneTest("i_3034f_geographic") }
+  @Test def test_i_3034f() { runner.runOneTest("i_3034f_military") }
+  @Test def test_i_3034f() { runner.runOneTest("i_3034f_utm_north") }
+  @Test def test_i_3034f() { runner.runOneTest("i_3034f_utm_south") }
```

Comment:
These def's all have the same name so there are duplicate method error when trying to run the tests. These should be named to match the test name (e.g. ``def test_i_3034f_decimal()``)

Reply:
fixing now...